package.json

```diff
@@ -23,7 +23,8 @@
     "node": ">=18"
   },
   "resolutions": {
-    "@sphereon/ssi-types": "0.30.2-feature.mdoc.funke2.367",
+    "@sphereon/ssi-types": "0.32.1-feature.VDX.341.53",
```

Comment:
just wondering if we will keep working with packages with branch-names, after merge to developer we might fix it

Reply:
I will take care of it, see other tickets